doc/tdshim_spec.md

```diff
@@ -762,6 +762,74 @@ The expected flow is:
    TDH.MR.EXTEND as described above, from the lowest GPA to the highest GPA,
    with 256 bytes granularity.
 
+#### Guideline
+
+TD Shim should follow [TCG Platform Firmware Profile specification](#tcg-pfp)
+in general.
+
+1. About separator
+
+   TD Shim shall measure 0x00000000 or 0xFFFFFFFF to RTMR[0~1] and the matching
+   EV_SEPARATOR event shall be recorded in the event log, prior to the
+   invocation of TD payload. As such, the verifier can know the rest events are
+   for the TD payload.
+
+   In case of an error occurs, TD shim shall measure 0x00000001 to RTMR[0~1] and
+   the matching EV_SEPARATOR event shall be recorded in the event log.
+
+2. About TD configuration
+
+   TD Shim shall extend the TD Hob to RTMR[0] and the event should be
+   EV_PLATFORM_CONFIG_FLAGS with event data being TD_SHIM_PLATFORM_CONFIG_INFO
+   and Descriptor being "td_hob\0", Info being the whole TD Hob.
+
+3. About payload info
+
+   TD Shim shall extend the payload binary to RTMR[1] and the event should be
+   EV_EFI_PLATFORM_FIRMWARE_BLOB2 with event data being
+   UEFI_PLATFORM_FIRMWARE_BLOB2 and BlobDescription being "td_payload\0".
+
+   TD Shim shall extend the payload parameter to RTMR[1] and the event should be
+   EV_PLATFORM_CONFIG_FLAGS with event data being TD_SHIM_PLATFORM_CONFIG_INFO
+   and Descriptor being "td_payload_info\0", Info being the whole TD Hob.
```

Comment:
`Info being the whole TD Hob` -> `Info being the whole payload parameters`. Right?